package.json

```diff
@@ -9,6 +9,7 @@
     "build-tauri": "tauri build",
     "dev-tauri": "tauri dev",
     "dev-browser-extension": "pnpm vite -c vite.config.chromium.ts",
+    "dev-firefox": "NODE_ENV=development pnpm vite build -c vite.config.firefox.ts --watch",
```

Comment:
any suggested name?

Reply:
How about change the dev-browser-extension to dev-chromium ？

Reply:
agree.